docs/release-process.md

```diff
@@ -0,0 +1,21 @@
+**Releasing a new version**
+
+Semanting versioning is used to release different versions of the action. Following steps are to be followed :
+
+1. Create a new branch for every major version. \
+Example, releases/v1, releases/v2.
+
+2. Cherry-pick the latest changes from main branch to the release branch.
```

Comment:
this point is more on patching a fix to existing release, we can remove the first line I think

Reply:
Updated it accordingly